Readme.md

```diff
@@ -1,73 +1,41 @@
+* Call the [RichEditExtension.Open](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.RichEditExtension.Open.overloads) method to open a new document with the specified document ID and content type, and retrieve the binary content from the passed model:
 
 ```cs
 @Html.DevExpress().RichEdit(settings => {
-    settings.Name = "RichEditNameHere";
-    settings.CallbackRouteValues = new { Controller = ..., Action = "ActionMethodThatHandlesRichEditCallbacks" };
-}).Open(UNIQUE_DOCUMENT_ID_HERE, RICH_TEXT_FORMAT_HERE, () => { return MODEL_BINARY_PROPERTY_HERE; }).GetHtml()
+    settings.Name = "RichEditName";
+    settings.CallbackRouteValues = new { Controller = "Home", Action = "RichEditPartial" };
+    //...
+}).Open(Model.DocumentId, Model.DocumentFormat, () => { return Model.Document; }).GetHtml()
 ```
+## Save a document
 
-<br>
-
-```vb
-@Html.DevExpress().RichEdit( _
-    Sub(settings)
-            settings.Name = "RichEditNameHere"
-            settings.CallbackRouteValues = New With {.Controller = "...", .Action = "ActionMethodThatHandlesRichEditCallbacks"}
-    End Sub).Open(UNIQUE_DOCUMENT_ID_HERE, RICH_TEXT_FORMAT_HERE, _
-              Function()
-                      Return MODEL_BINARY_PROPERTY_HERE
-              End Function).GetHtml()
-```
-
-<br>Save a document
-
-* Click the built-in toolbar's Save button/item.
-* Use the <a href="https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.RichEditSettings.Saving">RichEditSettings.Saving</a> property to handle the <a href="https://documentation.devexpress.com/AspNet/DevExpressWebOfficeDocumentManager_AutoSavingtopic.aspx">DocumentManager.AutoSaving</a> event.
-* Retrieve the modified content via the <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebMvcRichEditExtension_SaveCopytopic">RichEditExtension.SaveCopy</a> method, save it to the related bound model's binary property, and set the EventArgs Handled property to True (see the HomeController source code file):<br><br>
+* Click the Save ribbon command to initiate a save operation for the active document.
+* Use the [RichEditSettings.Saving](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.RichEditSettings.Saving) property to save a document in a byte array.
+* Call the [RichEditExtension.SaveCopy](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.RichEditExtension.SaveCopy.overloads) method to get the active document as a byte array, save it to the related bound model's binary property, and set the [Handled](https://docs.devexpress.com/AspNet/DevExpress.Web.Office.DocumentSavingEventArgs.Handled) property to `true` to prevent the default document processing:
```

Comment:
For my understanding, what do you mean by "the default document processing"?
Does this mean that the document will be processed in a different way?

Reply:
By default, the RichEdit tries to save the document by its path. But if a document is loaded from a byte array (like here), the control throws an exception, because Rich can not process such saving automatically. To prevent the exception, the Handled property should be set to `true`. This behavior is described in the following topic: [Save a Document](https://docs.devexpress.com/AspNet/403545/components/rich-text-editor/document-management/save-a-document). Perhaps we can add a link to this description:
```suggestion
* Call the [RichEditExtension.SaveCopy](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.RichEditExtension.SaveCopy.overloads) method to get the active document as a byte array, save it to the related bound model's binary property, and set the [Handled](https://docs.devexpress.com/AspNet/DevExpress.Web.Office.DocumentSavingEventArgs.Handled) property to `true` to prevent the [default document processing](https://docs.devexpress.com/AspNet/403545/components/rich-text-editor/document-management/save-a-document):
```

Reply:
Thank you for the clarification. Your suggestion looks fine. Please apply it.